### ISSUE TITLE
Maybe merge with 'official' Ansible for K3s

### ISSUE THREAD

**Issue by jfmatth** [PyratLabs/ansible-role-k3s#30]
https://github.com/rancher/k3s-ansible

Maybe contribute here and combine efforts

**Comment by xanmanning** [comment]
It's likely not possible to directly merge my code in as my project appears to take a slightly different approach to the official project. Also when I started this project (before the official project) I was building it with the ultimate aim of covering four phases in the lifecycle, "build" (deployment), "run" (operations post deployment) , "validation" (is it configured correctly, has configuration deviated from original build?) and "decommission" (tidying up when you're done).

I've really only made strides in "build" and some "run"/"decomission" phases, I'd like to move more into "validation".

I did not have motivation to work on the original project after it became more public as at the time it did not fulfill my needs, it seemed easier to advance my own code.

I'd be happy to merge my project with the official one, but I do this as a personal project and I don't have the same resources or following as Rancher.
